sdk/iotoperations/Azure.ResourceManager.IoTOperations/src/Azure.ResourceManager.IoTOperations.csproj

```diff
@@ -0,0 +1,8 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <PackageId>Azure.ResourceManager.IoTOperations</PackageId>
+    <Description>Azure Resource Manager client SDK for Azure resource provider IoTOperations.</Description>
+    <PackageTags>azure;management;arm;resource manager;ProviderNameLowercase</PackageTags>
```

Comment:
Same question here - if this should be the actual service name or not

Reply:
yes, you are right. I will quickly fix this. 
And this is a new repro [PR ](https://github.com/Azure/azure-sdk-for-net/pull/45916)with the issue you mentioned fixed.